### ISSUE TITLE
URL selection arrow and clickable area should be bigger

### ISSUE THREAD

**Issue by lilydjwg** [mozilla-mobile/fenix#22529]
### Steps to reproduce

1. type something in the address bar for a completion list from history
2. seeing that a desired item appears, try to click the bottom-left arrow to set address bar url to it

### Expected behaviour

The address bar url should be set from the item I clicked arrow and I can continue to edit it.

### Actual behaviour

The url is loaded directly.

### Device name

Sony Xperia XZ2 Compact

### Android version

Android 10

### Firefox release type

Firefox Nightly

### Firefox version

96.0a1 2021-11-21T17.44.25....

### Additional information

A lot of time I want to create a new url based on history one, e.g. Wikipedia and other MediaWiki sites, dictionary sites, or the repository page of a visited GitHub issue. I don't want to load the old url. It creates unwanted traffic and I need to click a few more times to edit the loaded url (and hope that that url doesn't redirect immediately).

**Comment by SoftVision-LorandJanos** [comment]
Hi @lilydjwg. Thank you for you report!
I couldn't reproduce the issue you described.
I tried to access an already accessed site where after I tapped on the arrow, the address was copied in the address bar, and there I could edit the URL with the desired modifications, and then it worked as expected.
Tested on the following builds:
- latest Nightly (2021-11-29).
- latest Beta (95.0.0-beta6).
- latest Release (94.1.2).

Device used: Xiaomi Mi11 Lite (Android11).

Please make a video of your issue if it still reproduces on the latest Nightly. Thank you!

**Comment by lilydjwg** [comment]
Emmm, I recorded a video, and found why: the clickable part of the arrow is smaller than I thought, and I was trying too hard to avoid clicking the url itself, clicking on the edge, which is out of the arrow's range.

So it actually works, but I think that action area should be larger anyway.

![图片](https://user-images.githubusercontent.com/440661/143877522-1fb25d91-3be5-4544-9715-ef129f34218b.png)

**Comment by SoftVision-LorandJanos** [comment]
In this case I can add a feature request label for improvement, and remove the bug label.

**Comment by Dunexus** [comment]
It should probably be labelled as a11y issue as small touch area issues usually are.

**Comment by mcarare** [comment]
For QA: You can also test this using the Accessibility Scanner on the search screen with suggestions.

**Comment by sflorean** [comment]
Tested on latest Beta 97.0.0-beta.6 and the following suggestion is made:
<img src="https://user-images.githubusercontent.com/58466444/151814157-114744ce-7039-4221-9852-83b02c95b4a7.jpg" width="270" height="550" />

**Comment by mcarare** [comment]
@sflorean can you re-test this on the latest nightly? I tested on the latest main and no suggestions are made. Thank you!

**Comment by lilydjwg** [comment]
My nightly doesn't show the arrow somehow. It reacts to touch but is empty visually.

**Comment by sflorean** [comment]
@mcarare I didn't posted results earlier from Nightly because the arrow was not displayed because of https://github.com/mozilla-mobile/fenix/issues/23378. 


On Nightly 98.0a1 - 2/3, no suggestions are made: 
<img src="https://user-images.githubusercontent.com/58466444/152305433-25d4f4e8-5a3e-4542-adc3-946815b8caf3.jpg" width="270" height="550" />

**Comment by mcarare** [comment]
Closing this as per the above comment.
